### ISSUE TITLE
When using Jaws screen reader and wet-boew error messages are not announced until navigating through the page

### ISSUE THREAD

**Issue by jorstor** [wet-boew/wet-boew#10059]
## Describe the bug

When error messages are present in the example using regular or specialized validation here https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-en.html jaws screen reader does not announce the errors but nvda screen reader does.

## To Reproduce

Steps to reproduce the behaviour:

1. Go to https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-en.html
2. Open jaws screen reader 
3. Click submit
4. Observe errors not being announced

## Expected behaviour

When an error is visible jaws screen reader should announce it automatically like nvda screen reader does.

## Who shall do the work?

I am asking for Principal publisher to please do the work

## Version of WET-BOEW/GCWeb you are using (optional)

v4.0.88.1

## Desktop/Smartphone (please complete the following information) (optional)

- Windows 11
- Chrome Version 141.0.7390.55
- Nvda screen reader 2024.4.1.34550
- Jaws screen reader 2025.2508.120

## Screenshots (optional)

<img width="1243" height="799" alt="Image" src="https://github.com/user-attachments/assets/d8ab7f98-9176-4f12-8dbe-190405069891" />

**Comment by EricDunsworth** [comment]
Might be related to #9853 (possibly the same underlying cause).
